lib/browser/init.ts

```diff
@@ -149,9 +150,6 @@ app.setPath('userData', path.join(app.getPath('appData'), app.getName()))
 app.setPath('userCache', path.join(app.getPath('cache'), app.getName()))
 app.setAppPath(packagePath)
 
-// Load the chrome devtools support.
```

Comment:
@MarshallOfSound seems like this was removed by mistake.

Reply:
Looks like a dodgy git rebase, I'll add it back